tests/util/defs.bzl

```diff
@@ -116,7 +116,7 @@ def _link_tree_impl(ctx):
     if prefix and not prefix.endswith('/'):
         prefix = prefix + "/"
     for link, target in ctx.attr.links.items():
-        print('  %s -> %s ' % (link, target))
+        # DBG print('  %s -> %s ' % (link, target))
```

Comment:
Did you intend to leave this?

Reply:
Yes. It is handy for printf debugging while trying to figure out what is going on. Thus the DBG